features/voorletters.feature

```diff
@@ -6,7 +6,7 @@ Functionaliteit: Bepalen van voorletters uit de voornamen van een persoon
 	Als een voornaam begint met een dubbelklank (Th, Ph, Ch, IJ, enz.), Dan wordt deze voornaam (ook) afgekort tot één voorletter.
 	Als één of meerdere voornamen uit één letter bestaan, dan volgt er na de letter geen "." (punt).
 	Wanneer na een voorletter zonder punt (voornaam had één letter) nog een andere voorletter volgt, wordt daartussen een spatie gezet.
-	Als de rubriek Voornamen is gevuld met de standaardwaarde '.' (punt), Dan wordt geen extra (scheidings)punt toegevoegd; de inhoud van de attribuut voorletters is na afleiding Dan '.'
+	Als voornamen enkel de standaardwaarde '.' (punt) bevat, dan wordt geen extra (scheidings)punt toegevoegd; de voorletters zijn dan '.' (punt).
```

Comment:
ik had deze gekopieerd van BRP (GBA). Komt dit ook voor in de BRK dat mensen een punt als voornamen krijgen?

Reply:
Dat weet ik zo niet, ik denk dat @kad-hebbim daar beter antwoord op kan geven dan ik.  
Ik denk wel dat het geen kwaad kan om dit in de specificatie op te nemen 🙂